working/0546-patterns/patterns-feature-specification.md

```diff
@@ -2054,11 +2087,18 @@ fail in some way.*
         expression's type is `dynamic`.* If it evaluates to `false`, continue to
         the next case (or default or exit).
 
-    3.  Execute the nearest non-empty case body at or following this case.
-        *You're allowed to have multiple empty cases where all preceding
-        ones share the same body with the last case.*
+    3.  Find the nearest non-empty case body at or following this case. *You're
+        allowed to have multiple empty cases where all preceding ones share the
+        same body with the last case.*
+
+    4.  Initialize any consistent variables declared in the body scope with the
+        values of the corresponding variables from the case scope. *There will
+        be no consistent variables and the body scope will be the case scope if
+        the body is only used by a single case.*
 
-    4.  Exit the switch statement. *An explicit `break` is no longer
+    5.  Execute the body statement.
+
+    6.  Exit the switch statement. *An explicit `break` is no longer
```

Comment:
"The switch statement completes normally."

(Using spec-terminology for "exiting".)

Actually, might need:

> 6. If execution of the body statement breaks without a label, then the switch statement completes normally.
> 7. If execution of the body statement continues with a label, and that label is labeling a
>     switch case of this `switch`, go to step 3 and continue from that label.
> 8. Otherwise the switch statement completes in the same way as the body statement.

(Or just leave this for the formal specification, and just be clear an understandable.)

Reply:
Done, but combined 6 with 8.